schemas/2017-08-01-preview/Microsoft.Genomics.json

```diff
@@ -45,15 +45,28 @@
         "etag": {
           "type": "string",
           "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {}
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Must exist in the request. Must not be null."
         }
```

Comment:
is this generated or hand-written? Is the swagger spec already updated?

Reply:
@Tiano2017 hand-written. We do not have a swagger spec. Is it required? Can you point me to some documentation that shows how this schema is being used and how it relates to the swagger spec? 

Also this PR is an attempt/best guess to resolve the problem described here: https://github.com/Azure/azure-resource-manager-schemas/pull/529#issuecomment-440423147

Thanks!